tracer.go

```diff
@@ -151,9 +153,8 @@ func (t *tracer) startSpanWithOptions(
 	operationName string,
 	options opentracing.StartSpanOptions,
 ) opentracing.Span {
-	startTime := options.StartTime
-	if startTime.IsZero() {
-		startTime = t.timeNow()
+	if options.StartTime.IsZero() {
```

Comment:
why test for this? If we are testing that FinishTime is zero, and StartTime is also zero, the failure of a startTime to be set will be apparent in the UI, is a visible (and ugly) bug.